README.md

```diff
@@ -341,6 +341,39 @@ interface {
 }
 ```
 
+#### [Instruct the controller to soft-reset (restart)](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=softreset)
+
+[compatible with schema version: 25+]
+
+```ts
+interface {
+  messageId: string;
+  command: "driver.soft_reset";
+}
+```
+
+#### [Instruct the controller to soft-reset (restart) only if soft-reset is enabled](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=softreset)
```

Comment:
Why do we nee this? Can't the client just catch the error and inspect it ?

Reply:
we don't, you are right that we can catch the error in the soft reset command. Only reason to keep it would be to match the driver API as closely as possible